resources/views/account/partials/address-form.blade.php

```diff
@@ -1,24 +1,24 @@
 <form slot-scope="{ variables, mutate, mutated }" v-on:submit.prevent="mutate">
-    <x-rapidez-ct::card.inactive>
+    <x-rapidez-ct::card.inactive class="mb-6">
```

Comment:
i dont think the mb-6 is needed to get spacing between the <card and the <toolbar, because the <x-sections component has that spacing within the component, can you see if you can just use that?